swift/apple/FirezoneKit/Sources/FirezoneKit/Features/SettingsView.swift

```diff
@@ -75,6 +75,116 @@ public final class SettingsViewModel: ObservableObject {
       }
     }
   }
+
+  func calculateLogDirSize(logger: Logger) -> String? {
+    logger.log("\(#function)")
+
+    let startTime = DispatchTime.now()
```

Comment:
I think we can probably remove the 1 minute refresh. It's just added complexity and code we need to maintain and test.

Who's viewing the log tab for more than a minute? If the user wants to refresh the logdir size this urgently they can just switch tabs back and forth.

Reply:
cc @ReactorScram

Reply:
It's unlikely that someone will figure out that we can switch tabs to refresh the log size number.

The timed refresh is arguably useful in the scenario where someone leaves the tab open and comes back after a long time, and then the number we're showing there might be very off the mark. Alternately, we could refresh when the app comes to the foreground while the tab is visible.

Reply:
> The timed refresh is arguably useful in the scenario where someone leaves the tab open and comes back after a long time,

I have a hard time imagining the likelihood of hitting this in reality. Also, the log file size probably wouldn't have changed much.

This tab will be used 99% of the time by us, and the remaining part of the time by a user who uses it to export or view the logs and then closes the window.

Is it really worth building this into the Android, Linux, and Windows clients too? If so, I think I'd need a stronger argument. If not, then we're heading down a path of inconsistent UX for each client.

Reply:
Okay, if that scenario is unlikely / unimportant, we don't need to refresh. I'll remove the refreshing part.

Reply:
Also, what if the size calculation takes longer than a minute? If the user is going to this tab it could be because the logdir size is extreme or contains tens of thousands of files.

Reply:
Ah you replied before my last comment, sounds good

Reply:
Remove refreshing in commit a994a8a.